refinery/tool_manager/views.py

```diff
@@ -164,3 +168,59 @@ def relaunch(self, request, *args, **kwargs):
         except Exception as e:
             logger.error(e)
             return HttpResponseBadRequest(e)
+
+
+class VisualizationToolProxy(Proxy, object):
```

Comment:
It was already a visualization-tool-proxy: Perhaps rename to reflect the difference? `AutoRelaunchProxy` or something like that.

Reply:
Would this be better left in the file where it's used? Or would that be weird?

Reply:
Agree with the renaming, I'd push to keep it in `views.py` since we're wrapping a view now